concepts/0011-decorators/README.md

```diff
@@ -1,7 +1,7 @@
 # Aries RFC 0011: Decorators
 
 - Authors: Daniel Hardman
-- Status: [DEMONSTRATED](/README.md#demonstrated)
+- Status: [ACCEPTED](/README.md#accepted)
```

Comment:
I think this may have leaked from another PR. Do we want to keep this in here, or wait to merge this until the Decorators status change occurs to merge it?

Reply:
Whoops! Yes, I'll take this out. Sorry about that.